### PR TITLE
Accept multiple pairs entered at once for binance exchange

### LINKED ISSUE #3425: Binance exchange settings: Accept multiple pairs entered at once
## Problem Definition

Due to the current limitations on importing past trades from binance (#3412), the only reasonable way to get something imported without running into timeouts is by maintaining an explicit allow list (https://discord.com/channels/657906918408585217/745172610240872508/884726268027875398).

The current UI accepts pairs entered one-by-one, offering an autocomplete drop-down that has to be clicked to add the matching pair from an API-queried list to the line edit. This works reasonably well for a handful of pairs, but especially when algorithmic trading is involved there might be 100+ pairs to be entered here. 

I expected to be able to batch-add a number of pairs by pasting a space-separated string with different pairs (e.g. "BTCUSDT ETHUSDT SOLUSDT"), but pressing "Save" only ends with ~~an endless progress bar that only stops when closing the app. In addition to that, the "Cancel" button is also disabled, making it impossible for the user to get out of the "in progress" state.~~ the overlay closing and discarding the entered list of pairs.

Edit: endless progress bar was from Binance API rate-limiting, disregard that.

### System Description

Operating system:  Win10
Rotki version: 1.20.1

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #3425 

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.
- [ ] Accept multiple pairs entered at once for binance exchange

### CODE REVIEW

**Comment by yabirgb** [comment]
So the functionality works okey. I'm failing to get it work but I believe is a backend issues. The pairs get correctly store in database

**Comment by lukicenturi** [comment]
> So the functionality works okey. I'm failing to get it work but I believe is a backend issues. The pairs get correctly store in database

What do you mean by "you're failing to get it work?" @yabirgb

**Comment by yabirgb** [comment]
> > So the functionality works okey. I'm failing to get it work but I believe is a backend issues. The pairs get correctly store in database
> 
> What do you mean by "you're failing to get it work?" @yabirgb

The pairs don't get correctly queried. I believe is related to the exchange initialization in the backend side but I need to investigate

**Comment by yabirgb** [comment]
So I would say technically the request of the issue is covered. The problem I had is because when the exchange gets added the trades/asset movements is queried with `cache_only=true` and the latest trades are not queried, but that is a different issue

File: frontend/app/tests/e2e/pages/history-page/trade-history-page.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export class TradeHistoryPage {
     return () => {
       // Wait for response.status to be 200
       cy.wait('@apiCall').its('response.statusCode').should('equal', 200);
+      cy.wait(500);
```
Comment (kelsos) [inline frontend/app/tests/e2e/pages/history-page/trade-history-page.ts:20]: Any other way to fix it than hardcoding a wait period?

File: frontend/app/src/components/helper/BinancePairsSelector.vue [hunk 2]
```diff
@@ -1,70 +1,68 @@
...
-        :class="outlined ? 'binance-market-selector--outlined' : null"
-        item-text="address"
-        item-value="address"
-        class="binance-market-selector"
-        :value="selection"
-        @input="input($event)"
-        @change="search = ''"
+  <v-autocomplete
+    v-bind="$attrs"
+    :items="allMakerts"
```
Comment (kelsos) [inline frontend/app/src/components/helper/BinancePairsSelector.vue:4]: Can we change this to `allMarkets`, it seems I missed that when we initially added the pair support

**Review by kelsos — APPROVED** [review]
lgtm

File: frontend/app/tests/e2e/pages/history-page/trade-history-page.ts (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/tests/e2e/pages/history-page/trade-history-page.ts:20]: I'm not sure, I can't find other easy way for fixing this. The problem is sometimes, the order of flow is like this:
call api -> cypress get some element -> the api changes is refresh whole table -> the previous element that cypress get is gone now.

File: frontend/app/tests/e2e/pages/defi-page.ts [hunk 3]
```diff
@@ -19,9 +19,14 @@ export class DefiPage {
...
-      cy.get(`#defi-module-${module}`).find('button').click();
-      cy.get(`#defi-module-${module}`).should('not.exist');
+      const moduleId = `#defi-module-${module}`;
+      cy.get('body').then($body => {
+        if ($body.find(moduleId).length > 0) {
+          cy.get(moduleId).scrollIntoView();
+          cy.get(moduleId).find('button').click();
+          cy.get(moduleId).should('not.exist');
+        }
+      });
```
Comment (kelsos) [inline frontend/app/tests/e2e/pages/defi-page.ts:29]: This change should be reverted I and instead, we should remove the entries for the new modules that are now deactivated by default

**Pull request merged**